src/constants.ts

```diff
@@ -38,4 +38,7 @@ export const DEBUG_ADDRESSES = (process.env.DEBUG_ADDRESSES || '')
   .split(',')
   .filter(isEthereumAddress)
   .map((address) => address.toLowerCase())
-export const SNAPSHOT_STATUS_ENABLED = false
+
+export const SNAPSHOT_STATUS_ENABLED = process.env.SNAPSHOT_STATUS_ENABLED === 'true'
```

Comment:
You can use `getBooleanStringVar`

Reply:
I thought about it but it uses a clientEnv method that takes the var from config.json, I wasn't sure that I could use it in this case

Reply:
We can check this in another PR

Reply:
https://github.com/decentraland/governance/issues/1328